spec/01-unit/010-router_spec.lua

```diff
@@ -95,14 +95,14 @@ describe("Router", function()
 
     it("[uri]", function()
       -- uri
-      local match_t = router.select("GET", "/my-api")
+      local match_t = router.select("GET", "/my-api", "domain.org")
```

Comment:
traditionally (and per IANA recommendations) we should probably use "example.com" or "example.org". it's not a _huge_ difference, but historically usage of things like "domain.com" has been problematic (that domain was used to serve malware at one point).